frontend-react/src/components/ManagePublicKey/ManagePublicKeyUpload.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useRef } from "react";
+import {
+    Button,
+    FileInput,
+    FileInputRef,
+    Form,
+    FormGroup,
+    Label,
+} from "@trussworks/react-uswds";
+
+export interface ManagePublicKeyFormProps {
+    onPublicKeySubmit: (e: React.FormEvent<HTMLFormElement>) => void;
+    onFileChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    onBack: () => void;
+    fileName: string;
+}
+
+const BASE_ACCEPT_VALUE = ".pem";
+
+export const ManagePublicKeyUpload = ({
+    onPublicKeySubmit,
+    onFileChange,
+    onBack,
+    fileName,
```

Comment:
Shouldn't this receive a File instance instead?

Reply:
Will update